bin/linters.sh

```diff
@@ -82,6 +82,7 @@ go_metalinter() {
         --enable=vet\
         --enable=vetshadow\
         --skip=testdata\
+        --skip=vendor\
         --vendor\
         $PKGS
 }
```

Comment:
i could be mistaken, but I was under the impression that `buildifier` only worked on `BUILD` files and could not be run on skylark files.